packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -116,6 +117,11 @@ abstract class FlameChartState<T extends FlameChart, V> extends State<T>
   // for an explanation of how we previously zoomed from level 2.0 to level 3.0.
   double get wasdZoomOutUnit => zoomController.value * 1 / 3;
```

Comment:
not from this cl but maybe change this to keyboardZoomOutUnit as the key concept is keyboard navigation not the specific keys used.

Reply:
done.